recipes/cmocka/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "cmocka" %}
+{% set version = "1.1.5" %}
+{% set version_major_minor = ".".join(version.split('.')[:2]) %}
+{% set so_version = "0.7.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://cmocka.org/files/{{ version_major_minor }}/{{ name }}-{{ version }}.tar.xz
+  sha256: f0ccd8242d55e2fd74b16ba518359151f6f8383ff8aef4976e48393f77bba8b6
+
+build:
+  number: 0
+  run_exports:
+    # No correlation between library name and
+    # so names
+    # https://gitlab.com/cmocka/cmocka/-/issues/43
+    - {{ pin_subpackage(name, max_pin='x.x.x') }}
```

Comment:
Do i need to create a package `libcmocka` with the `so_version` so I can pin to that correctly?

Seems tedious to have to update that number myself everytime.

Reply:
No, no need

Reply:
alright, then i'm good with this as is.